python/cudf/cudf/_lib/groupby.pyx

```diff
@@ -240,6 +242,10 @@ def _drop_unsupported_aggs(Table values, aggs):
         elif (
                 is_decimal_dtype(values._data[col_name].dtype)
         ):
+            if rmm._cuda.gpu.runtimeGetVersion() < 11000:
+                raise RuntimeError(
+                    "Decimal aggregations are not supported on CUDA 10.x."
```

Comment:
Can we reword this to say that it's only supported on CUDA >= 11.0 due to an nvcc compiler bug? Right now it feels arbitrary and like we're forcing users to upgrade without reason.